priv/repo/migrations/20170509093107_add_likes_table.exs

```diff
@@ -0,0 +1,11 @@
+defmodule App.Repo.Migrations.AddLikesTable do
+  use Ecto.Migration
+
+  def change do
+    create table :likes do
+      add :user_hash, :string
+      add :article_id, :integer
+      add :like_value, :integer
```

Comment:
I followed what @nelsonic wrote for me on storing these. I think either option is pretty clear